src/index.css

```
@@ -12,15 +12,13 @@ body {
   background-color: #E5E5E5;
 }
 
-li {
-  list-style: none;
-}
-
-input {
+input{
   outline: none;
   border: none;
 }
 
+a{
+=======
 a {
```

This conflict should be resolved.
```suggestion
a {
```